platform/containers/prometheus/run.sh

```diff
@@ -13,6 +13,9 @@ set -ex
     exit 1
 fi
 
+# Clear previous data in the directory
```

Comment:
Nice. I guess this is how we work around when a program takes more memory during crash recovery than normal running.
It'll be nice if we can add a flag to indicate whether this is restart due to oom or others. If we can keep data for other restarts, that'll be nice.